net/instaweb/rewriter/rewrite_query.cc

```diff
@@ -545,7 +545,7 @@ RewriteQuery::Status RewriteQuery::ScanNameValue(
       if (pairs[i] == "no-transform") {
         // TODO(jmarantz): A .pagespeed resource should return un-optimized
         // content with "Cache-Control: no-transform".
-        options->set_enabled(RewriteOptions::kEnabledOff);
+        options->set_enabled(RewriteOptions::kEnabledStandby);
```

Comment:
What are you concerned about?  If we get as far as interpreting query parameters then we're definitely not unplugged, so standby seems right to me,.